src/Sentry.Unity.Android/SentryJava.cs

```diff
@@ -40,6 +43,93 @@ internal class SentryJava : ISentryJava
 {
     private static AndroidJavaObject GetSentryJava() => new AndroidJavaClass("io.sentry.Sentry");
 
+    public bool IsEnabled(IJniExecutor jniExecutor)
+    {
+        return jniExecutor.Run(() =>
+        {
+            using var sentry = GetSentryJava();
+            return sentry.CallStatic<bool>("isEnabled");
+        });
+    }
+
+    public bool Init(IJniExecutor jniExecutor, SentryUnityOptions options)
+    {
+        jniExecutor.Run(() =>
+        {
+            using var sentry = new AndroidJavaClass("io.sentry.android.core.SentryAndroid");
+            using var context = new AndroidJavaClass("com.unity3d.player.UnityPlayer")
+                .GetStatic<AndroidJavaObject>("currentActivity");
+
+            sentry.CallStatic("init", context, new AndroidOptionsConfiguration(androidOptions =>
```

Comment:
Curious if this affects the startup time of a game in any significant way. Since now we're doing this via P/Invoke what used to be direct

Reply:
From local testing, `Init` takes about 25ms. Based on what I've got from the Android SDK itself, self-init takes around that time dring self-initialization as well.
So we just moved the time from "before Unity starts" to "while Unity starts" and the P/Invoke does not add any significant overhead to initialization.